Services/Catalogue/Mappers/ConfigurableProductsData.php

```diff
@@ -261,11 +264,28 @@ private function getCategoryArray($category)
          'image' => ($category->getImageUrl()) ? $category->getImageUrl() : '',
          'url' => $category->getUrl(),
          'isActive'=> $category->getIsActive(),
-         'includeInMenu' => ($category->getIncludeInMenu()) ? true : false,
          'pathIds' => $pathIds,
-         'isSearchable' => (!$category->getIncludeInMenu() || !$category->getIsActive()) ? false : true,
          'parentId' => ($category->getParentCategory()) ? $category->getParentCategory()->getId() : '',
          'parentUrlKey' => ($category->getParentCategory()) ? $category->getParentCategory()->getUrlKey() : '',
         ];
+
+        $includeInMenu = false;
```

Comment:
By default set two variables based on conditions we have set in the array itself and modify it based on categories ignore logic.